app/helpers/invitation_codes_helper.rb

```diff
@@ -6,13 +6,13 @@ def invite_hidden_tag(invite)
   end
 
   def invite_link(invite_code)
-    text_field_tag :invite_code, invite_code_url(invite_code), :readonly => true
+    text_field_tag :invite_code, invite_code_url(invite_code), :class => 'form-control', :readonly => true
```

Comment:
I think for now adding them to every field is the best solution, I'm not too fond of overwriting the rails helper. The midterm solution would be to switch to something like simple_form which should have configuration options for that.